packages/next/src/server/config.ts

```diff
@@ -807,6 +807,18 @@ function assignDefaults(
     }
   }
 
+  if (result.experimental?.cacheLife) {
+    const defaultCacheLifeProfile = result.experimental.cacheLife['default']
+    if (!defaultCacheLifeProfile) {
+      throw new Error('No default cacheLife profile.')
```

Comment:
Should we just use our default one if one isn't provided instead of erroring?

Reply:
Well the idea is that it should be resolved when normalizing the cache and so it something would've gone wrong while normalizing.

Reply:
I updated the normalization logic to always fill in the blanks.